pkg/ccl/backupccl/backup_processor.go

```diff
@@ -85,13 +85,15 @@ var (
 		"bulkio.backup.split_keys_on_timestamps",
 		"split backup data on timestamps when writing revision history",
 		true,
+		settings.WithName("bulkio.backup.split_keys_on_timestamps.enabled"),
```

Comment:
@dt Now that we have the option to rename things, I wonder if we want to make a follow-up pass to @knz's work here to address some of our long-standing setting naming gripes?